### ISSUE TITLE
[Feature request] Close RapidCRC after "Put CRC in Filename"

### ISSUE THREAD

**Issue by danielmmmm** [OV2/RapidCRC-Unicode#174]
It would be very handy if the UI would terminate itself after completion of "Put CRC in Filename". In case of an error during the operation, the UI should stay running, of course.

**Comment by vatterspun** [comment]
So current behavior is of course that you have to close the program after any operation.  Since it sounds like you do a lot of CRC-in-filename work, it sounds like an auto-close would be easier.  So you're asking for is like a checkbox to terminate the program after running?

If the dev isn't interested, you might want to look into CLI options.

**Comment by danielmmmm** [comment]
Yes, a setting or a checkbox that stays checked when the program exits would work fine.
The CLI option that [mugiritsu](https://github.com/mugiritsu) proposed in [https://github.com/OV2/RapidCRC-Unicode/issues/173](https://github.com/OV2/RapidCRC-Unicode/issues/173) would work perfectly fine too.

And, of course, I won't cry (out loud) if the dev decides not to implement a setting/checkbox or CLI option. RapidCRC is incredibly useful as is :-)

**Comment by danielmmmm** [comment]
Oh, this looks promising, thanks a lot!!!
